cli.js

```diff
@@ -1,84 +1,132 @@
-const commander = require('commander');
-// grab the version to keep things consistent
-const { version } = require('./package.json');
+const yargs = require('yargs');
+const moniker = require('moniker');
+
+const logger = require('./lib/logger');
+const { makeNameValid } = require('./lib/nameUtil');
 const { deployHandler, createHandler, invokeHandler,
-  logHandler, loginHandler, removeHandler, unknownHandler } = require('./lib');
+  logsHandler, loginHandler, removeHandler } = require('./lib');
 
-const actionWrapper = function actionWrapper(action, actionOptions) {
-  return async (options) => {
-    await action({
-      options: Object.assign({}, actionOptions, options, { path: commander.path }),
-      args: commander.args,
-      rawArgs: commander.rawArgs,
-      name: commander.args[commander.args.length - 1]._name,
-    });
-  };
-};
+const handleCommand = async function handleCommand(options, specificHandler) {
+  const numArgs = options._.length;
+  if (numArgs > 1) {
+    yargs.showHelp();
+    logger.error('Too many positional args given for command');
```

Comment:
This is a user-facing message? Because if so it has to be cleaner.

Reply:
It is, I would love to change it to something more meaningful and succinct. Any suggestions?